### DISCUSSION TITLE
Bridge problem

### DISCUSSION THREAD

**Discussion by twisterdw** [ethereum-optimism/developers#276] [General]
### Discussion Topic

hey.i bridged my funds from op to eth mainnet in 2021 now it shows that my funds are on op but i dont see it on my wallet and when i try to bridge it initiate withdrawal button does not work.i tryed 100 times to witdraw since 2021 september but it stucked.can somebody help?opeden 1000 times tickets on discord but team said that i must to wait.whats wrong with you guys?i want want to claim my funds.i dont see it on metamask i see these funds only on gateway.optimism web

**Comment by imtipi** [comment]
can you try withdraw on https://superchainrelayer.xyz/ (github repo https://github.com/ethereum-optimism/superchain-relayer)?
if can't,can you provide your txid here?

**Reply by twisterdw** [reply]
Sure  this is my txid.its impossible to find it on explorer or relayer

0x0f2f70ce16c2739a1d9e1220e075c3d2b4214e105db0390e3afe4f7370af0109

**Reply by twisterdw** [reply]
I see funds od gateway optimism but not in my wallet.transfer was made in 2021.08.17

**Reply by twisterdw** [reply]
![Screenshot_2024-03-10-19-11-02-157_io metamask](https://github.com/ethereum-optimism/developers/assets/34404784/5250a14f-bb3c-4146-9c71-dfcf34616bc7)

**Reply by twisterdw** [reply]
But cant withdraw these funds from app.optimism web

**Reply by imtipi** [reply]
cc @sbvegan
